moped-database/views/project_list_view.sql

```diff
@@ -201,4 +191,9 @@ AS WITH project_person_list_lookup AS (
     mpps.name,
     cpl.children_project_ids,
     proj_notes.project_note,
-    proj_notes.date_created;
+    proj_notes.date_created,
+    work_activities.contractors,
+    work_activities.contract_numbers,
+    work_activities.task_order_names,
+    work_activities.task_order_names_short,
```

Comment:
oops—i lied: one more part of the diff is that i decided to introduce a `task_order_names_short` field that is just ID part of the task order, e.g. `24ART81006`.